test/e2e/space_autocompletion_test.go

```diff
@@ -148,16 +134,13 @@ func TestAutomaticClusterAssignment(t *testing.T) {
 
 		t.Run("after both members marking as full then the new space won't be provisioned", func(t *testing.T) {
 			// given
-			var memberLimits []testconfig.PerMemberClusterOptionInt
 			for _, m := range toolchainStatus.Status.Members {
 				if memberAwait1.ClusterName == m.ClusterName {
-					memberLimits = append(memberLimits, testconfig.PerMemberCluster(memberAwait1.ClusterName, m.SpaceCount))
+					spaceprovisionerconfig.UpdateForCluster(t, hostAwait.Awaitility, memberAwait1.ClusterName, testSpc.MaxNumberOfSpaces(uint(m.SpaceCount)), testSpc.Enabled(true))
 				} else if memberAwait2.ClusterName == m.ClusterName {
-					memberLimits = append(memberLimits, testconfig.PerMemberCluster(memberAwait2.ClusterName, m.SpaceCount))
+					spaceprovisionerconfig.UpdateForCluster(t, hostAwait.Awaitility, memberAwait2.ClusterName, testSpc.MaxNumberOfSpaces(uint(m.SpaceCount)), testSpc.Enabled(true))
 				}
 			}
```

Comment:
this could be probably simplified to 
```
for _, m := range toolchainStatus.Status.Members {
	spaceprovisionerconfig.UpdateForCluster(t, hostAwait.Awaitility, m.ClusterName, testSpc.MaxNumberOfSpaces(uint(m.SpaceCount)), testSpc.Enabled(true))
}
```